app/views/users/_userid.html.erb

```diff
@@ -0,0 +1,30 @@
+<div class="container">
+<div class="user_profile">
+<div class="photos">
+    <ul class="photo"> 
+   <li> <img src="<%= @user.photo %>"> </li>
+   </ul>
+</div>
+<div class="names_posts">
+    <ul class="name_post"> 
+   <li class="list_name"> <%= @user.name %> </li>
+   <li class="list_countpost">Number of post: <%= @user.postscounter %> </li>
+   </ul>
+</div>
+</div>
+<div class="bio">
+    <ul class="bio_info width"> 
+    <h3 class="title">Bio</h3>
+   <li class="bio_content"> <%= @user.bio %> </li>
+   </ul>
+</div>
+<div class="post">
+    <ul class="post_info width"> 
+    <h3 class="title"><%= @post.title%> #<%= @post.id%></h3>
+   <li class="bio_content"> <%= @post.text %> </li>
+   <li class="list_countcomentlikes">Comments: <%= @post.commentscounter %>, Likes: <%= @post.likescounter %> </li>
+   </ul>
+</div>
+<button class="allcoment"> See all posts</button>
```

Comment:
- [ ] Please add the link to display all user post in your `user/user_show.html.erb` file
```ruby
<button>
<%= link_to 'See all Post', user_posts_path(@user) %>
</button>
```